stock_request/README.rst

```diff
@@ -80,7 +80,7 @@ also cancelled.
 Known issues / Roadmap
 ======================
 
-There is no way to achieve Storck Request and Stock Request Orders.
+There is no way to archive Stock Request and Stock Request Orders.
```

Comment:
@abbasEbadian Please change this in readme fragments instead.

Reply:
Done.